src/LibraryManager/Configuration/Settings.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Xml;
+using System.Xml.Linq;
+using Microsoft.Web.LibraryManager.Contracts.Configuration;
+using Microsoft.Web.LibraryManager.Utilities;
+
+namespace Microsoft.Web.LibraryManager.Configuration
+{
+    internal class Settings : ISettings
+    {
+        /// <summary>
+        /// File name where user settings (not project settings) are stored
+        /// </summary>
+        private const string ConfigFileName = "libman.config";
+
+        private XDocument _configFile;
+
+        // protected for tests to override
+        protected virtual string ConfigFilePath => Path.Combine(UserDataRoot, ConfigFileName);
```

Comment:
I'd still at least save the result of the first ConfigFilePath in a local variable in the Settings constructor since it gets called in there a few times, but I'll leave it up you.